WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -333,28 +331,36 @@ extension MainTabBarController {
 }
 
 
-// MARK: - Tab dot madness!
+// MARK: - Reviews Tab Badge Updates
```

Comment:
I'm introducing a `ReviewsCoordinator` in a future PR to hopefully encapsulate some of these. Hopefully. 😄 

It looks like we started using Coordinators but it never took off. 

https://github.com/woocommerce/woocommerce-ios/blob/eec8ec29b4cfc8e215ecd1550d6f5ee434d07dcd/WooCommerce/Classes/Tools/Coordinator.swift#L4-L12

Reply:
sounds like a great idea! the navigation code is now scattered in the view controllers. it would be cool to try using Coordinators 👍